rakefile

```diff
@@ -398,3 +398,19 @@ RSpec::Core::RakeTask.new(:spec) do |t|
   t.rspec_opts = ['--color']
   t.pattern = 'spec/'
 end
+
+desc 'Clean up artifacts created by performance runs'
+task :clean do
+  rm_rf "simulation-runner/cache"
+  rm_rf "simulation-runner/config/tmp"
+  rm_rf "gatling-recorder.log"
+  rm_rf "cache"
+  rm_rf "log"
+  rm_rf "junit"
+  rm_rf "last_abs_resource_hosts.log"
+  rm_rf "log"
+  rm_rf Dir.glob('results/perf/PERF_*').grep(/PERF_[0-9]+.*$/)
```

Comment:
you can also have results/scale/...  Not sure the names you will see there though.  @billclaytor would have them on hand.

Reply:
Thanks @RandellP that value is defined at https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/tests/helpers/perf_run_helper.rb#L61
I have amended the commit to include the removal of the `results/scale/PERF_SCALE_*` glob.